src/plugins/Display/Display_Mono_128X64.h

```diff
@@ -9,93 +9,81 @@
 class DisplayMono128X64 : public DisplayMono {
     public:
         DisplayMono128X64() : DisplayMono() {
-            mEnPowerSafe = true;
+            mEnPowerSave = true;
             mEnScreenSaver = true;
             mLuminance = 60;
             mDispY = 0;
             mTimeout = DISP_DEFAULT_TIMEOUT;  // interval at which to power save (milliseconds)
-            mUtcTs = NULL;
-            mType = 0;
         }
 
-        void init(uint8_t type, uint8_t rotation, uint8_t cs, uint8_t dc, uint8_t reset, uint8_t clock, uint8_t data, uint32_t *utcTs, const char *version) {
+        void config(bool enPowerSave, bool enScreenSaver, uint8_t lum) {
+            mEnPowerSave = enPowerSave;
+            mEnScreenSaver = enScreenSaver;
+            mLuminance = lum;
+        }
 
+        void init(uint8_t type, uint8_t rotation, uint8_t cs, uint8_t dc, uint8_t reset, uint8_t clock, uint8_t data, DisplayData *displayData) {
             u8g2_cb_t *rot = (u8g2_cb_t *)((rotation != 0x00) ? U8G2_R2 : U8G2_R0);
-            mType = type;
-
             switch (type) {
                 case 1:
-                    mDisplay = new U8G2_SSD1306_128X64_NONAME_F_HW_I2C(rot, reset, clock, data);
+                    monoInit(new U8G2_SSD1306_128X64_NONAME_F_HW_I2C(rot, reset, clock, data), type, displayData);
                     break;
-                default:
                 case 2:
-                    mDisplay = new U8G2_SH1106_128X64_NONAME_F_HW_I2C(rot, reset, clock, data);
+                default:
+                    monoInit(new U8G2_SH1106_128X64_NONAME_F_HW_I2C(rot, reset, clock, data), type, displayData);
                     break;
             }
-
-            mUtcTs = utcTs;
-
-            mDisplay->begin();
             calcLinePositions();
-
-            mDisplay->clearBuffer();
-            mDisplay->setContrast(mLuminance);
-            printText("AHOY!", 0, 35);
+            printText("AHoy!", 0, 35);
```

Comment:
Wurde diese Zeile extra "umgeschrieben"? **AH**oy

Reply:
Ja ich dachte eigentlich, in dieser Form stünde es auch auf der Homepage, aber ich sehe gerade, das was ein Irrtum. Ahoy! wäre korrekt.